### DISCUSSION TITLE
Focalpoints not invalidated

### DISCUSSION THREAD

**Discussion by ghost** [spatie/statamic-responsive-images#238] [Q&A]
Hey,

I have an issue where the half-strategy cache get's not invalidated when changing the focal point of an asset.

It works, when the asset is changed or replaced.
I'm using the Responsive Images Plugin Version 4.1.1 to display assets and Statamic Pro 4.23.2.
Even when I'm setting the invalidation rules in the static_caching.php to rules' => 'all, it only refreshes the image when replaced or changed. Otherwise I have to either delete the static cache manually in the control-panel or i have to clear my browser cache.
Both options are not convenient for normal users and not usable for websites which modify their assets quite often.

Do you have any idea where to look?

Or can someone check with static caching, invalidation rules and responsive image plugin with this repro steps?

1. Add an image in the Statamic Control Panel like a portrait of a person
2. Save, publish and check on page if it's displayed
3. Open the asset in the CP and set a zoom focal point onto the face e.g.
4. Save, publish and check on page if it's displayed
5. For me, it's not. And also not changes in the focal point once it's displayed due to forced refresh

This is the usual implementation of our images:

```
{{ employees:portrait }}
    {{ responsive src="{url}" alt="{title}" class="employee-portrait" width="800" height="500" ratio="3/2" }}
{{ /employees:portrait }}
```

**Comment by ncla** [comment]
Can you please try this same thing but by replacing `responsive` tag with regular `glide` tag? Depending on the outcome, this may be bug in this addon or in CMS itself.

**Reply by ghost** [reply]
Good point. I replaced the `responsive` with the `glide` tag - Still the same issue. I'm gonna report it in the cms repo.

Thanks!

**Comment by ghost** [comment]
The issue in the CMS repo can be found [here](https://github.com/statamic/cms/issues/8856)
